.eslint-bin/pre-commit-hook.js

```diff
@@ -12,6 +12,7 @@ function getFilesModified(callback) {
     }
 
     listFilesModified = status.files
+      .filter(change => change.index !== 'D')
```

Comment:
Actually it's even more than that, it would be nice to remove the file from the list file as it is useless now

Reply:
? It is what it do, isn't it?
The added line do: filter files, do not take deleted ones

Reply:
Et les tests passent pas